api/src/pcapi/routes/public/collective/endpoints/simulate_adage_steps/utils.py

```diff
@@ -0,0 +1,16 @@
+from functools import wraps
+import typing
+
+from pcapi import settings
+
+from .errors import UnauthorizedEnvironment
+
+
+def exclude_prod_environment(func: typing.Callable) -> typing.Callable:
+    @wraps(func)
+    def decorator(*args: typing.Sequence, **kwargs: typing.Mapping) -> typing.Any:
```

Comment:
Alors si on chipote ça serait plutôt `decorated_function` car le décorateur c'est `exclude_prod_environment`
```suggestion
    def decorated_function(*args: typing.Sequence, **kwargs: typing.Mapping) -> typing.Any:
```